ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,67 @@
+<!--
+
+Hi there! Here's a few pointers to help us help you with your issue as quickly
+as possible.
+
+We prefer that the RSpec Rails issue tracker be mainly used for bug reports.
+Feature requests or general requests for help should go to our google group:
+rspec@googlegroups.com. We use that as our primary location for higher level
+RSpec discussions.
+
+If you are filing a bug report, there's a few steps it'd be super if you could
+follow. If you can't do all of these, don't worry, but if you can it'll help us
+resolve your bug faster!
+
+-->
+
+## Observed behaviour
+
+<!--
+please provide a concise description of the behaviour you are observing with
+RSpec and Rails
+-->
+
+## Expected behaviour
+
+<!--
+Please provide a description of what you expect to be happening, and how that
+differs from the current behaviour.
+-->
+
+## Can you provide an example app?
+
+<!--
+This step is probably the most important in allowing us to quickly debug
+your issue, and we typically ask that you follow these steps:
+
+1. `rails new` an app at the specific version of ruby and rails that you are
+   using
+2. commit that app, so that we have the rails skeleton in a separate commit
+3. Make all the changes necessary (adding RSpec, models, specs, controllers,
+   etc) to reproduce the issue. There should be a failing test or tests that you
+   expect to pass. We'll run `bundle exec rspec`, but if you can provide a
+   readme with more detailed instructions that'd be amazing :)
+4. Commit
+5. Provide a link to a github repo, a description of the app and what you're expecting here
+-->
+
+## What Ruby, Rails and RSpec versions are you using?
+
+<!--
+
+You can run
+
+```
+bundle exec ruby --version
+bundle exec rails --version
+bundle exec rspec --version
+```
+
+and paste the output below
+
+-->
+
+## Did this problem exist before you upgraded to Rails 5?
```

Comment:
if you want a freeform answer you might widen this to "did you notice this bug after a specific update or event? please explain. e.g. updated to rails 5"